### ISSUE TITLE
ColorPicker changes the front color, but need to change background color

### ISSUE THREAD

**Issue by geohuz** [mikbry/material-ui-color#67]
Using the latest version of material-ui, simply put the Colorpicker component into use and change the color , but the button color doesn't get changed, when using the inspector and found that only background color changing will be reflected on the button color.

**Comment by mikbry** [comment]
I made an example here:
https://codesandbox.io/s/material-ui-color-picker-qvcg3
Could you share the code you are using ?

**Comment by geohuz** [comment]
Hi, here's my code,

```
 <ColorPicker defaultValue="red" />
```

I've checked my code with yours demo, I don't use theme provider, cssbaseline, but even after adding these components I still get button's color changed(not the background color).  I do noticed that I'm using the version   "@material-ui/core": "^4.10.1".

Finally I found the issue, that's the latest reactjs version causes issue, which is "16.13.1"!
